README.md

```diff
@@ -68,13 +72,13 @@ status:
       type: ethernet
       ipv4:
         address:
-        - ip: 192.168.66.101
+        - ip: 192.0.2.101
           prefix-length: 24
         dhcp: false
         enabled: true
       ipv6:
         address:
-        - ip: fe80::5055:ff:fed1:5501
+        - ip: 2001:db8:1:5001
           prefix-length: 64
         autoConf: false
```

Comment:
A general question: Is this supported at the moment here? dhcp?
I remember only a few options (like mtu) have been supported, everything else is not.
If it is not, we better take it out and put here only tested stuff.

https://github.com/nmstate/nmstate/blob/master/libnmstate/schemas/operational-state.yaml#L337

Reply:
So what has been implemented? I see ton of stuff in the link. Should I drop the whole ipv6 section?